--- corpora/europeana/ner-corpora-master/enp_DE.sbb.bio/enp_DE.sbb.bio	2019-05-15 01:28:56.000000000 +0200
+++ corpora/europeana/enp_DE.sbb.bio.fixed	2022-04-21 15:13:04.901173994 +0200
@@ -1506,7 +1506,8 @@
 neuer O
 Roman O
 „ O
-Die Böhminger O
+Die O
+Böhminger O
 ' O
 dürfte O
 in O
@@ -2729,9 +2730,13 @@
 „ O
 „ O
 Mittagskost O
-... O
+. O
+. O
+. O
 43 O
-„ . . O
+, O
+. O
+. O
 38 O
 c. O
 „ O
@@ -2746,10 +2751,13 @@
 „ O
 „ O
 Morgenkost O
-. . . O
+. O
+. O
+. O
 16 O
 „ O
-. . O
+. O
+. O
 11 O
 „ O
 * O
@@ -3511,89 +3519,89 @@
 erbeten O
 hätten O
 . O
-Bezüglich  O
+Bezüglich O
 Saiut B-PER
 Valliers I-PER
-höre  O
-ich  O
-aus  O
-bester  O
+höre O
+ich O
+aus O
+bester O
 Quelle O
-,  O
-daß  O
-derselbe  O
-auf  O
+, O
+daß O
+derselbe O
+auf O
 Waddingtons B-PER
-heute  O
-an  O
-ihn  O
-gerichtetes  O
-Schreiben  O
-zur  O
-Zurücknahme  O
-seiner  O
-Demission  O
-veranlaßt  O
-werden  O
+heute O
+an O
+ihn O
+gerichtetes O
+Schreiben O
+zur O
+Zurücknahme O
+seiner O
+Demission O
+veranlaßt O
+werden O
 dürfte O
-.  O
+. O
 Saiut B-PER
 Ballier I-PER
-hatte  O
-wiederholt  O
-erklärt  O
-,  O
-sich  O
-von  O
+hatte O
+wiederholt O
+erklärt O
+, O
+sich O
+von O
 Waddington B-PER
-nicht  O
-trennen  O
-,  O
-sondern  O
-mit  O
-diesen  O
-stehen  O
-und  O
-fallen  O
-zu  O
-wollen  O
-.  O
-Der  O
-abgegangene  O
-Minister  O
-hat  O
-nunmehr  O
-den  O
-Botschafter  O
-förmlich  O
-seines  O
-Wortes  O
-entbunden  O
-.  O
-Der  O
-diplomatische  O
-Eisgang  O
-hat  O
-übrigens  O
-begonnen  O
-:  O
-Herr  O
-v  O
-.  O
+nicht O
+trennen O
+, O
+sondern O
+mit O
+diesen O
+stehen O
+und O
+fallen O
+zu O
+wollen O
+. O
+Der O
+abgegangene O
+Minister O
+hat O
+nunmehr O
+den O
+Botschafter O
+förmlich O
+seines O
+Wortes O
+entbunden O
+. O
+Der O
+diplomatische O
+Eisgang O
+hat O
+übrigens O
+begonnen O
+: O
+Herr O
+v O
+. O
 Pontecoulant B-PER
-,  O
-als  O
-Kabinetsdirektor  O
-seit  O
-fünfzehn  O
-Jahren  O
-an  O
-der  O
-Spitze  O
-des  O
+, O
+als O
+Kabinetsdirektor O
+seit O
+fünfzehn O
+Jahren O
+an O
+der O
+Spitze O
+des O
 auswärtigen B-ORG
 Amtes I-ORG
-,  O
+, O
 ist O
 ausgeschieden O
 . O
@@ -4030,7 +4038,8 @@
 Ungarische O
 Goldrente O
 83'/ O
-, (L3> O
+, O
+(L3> O
 , O
 bvroz O
 . O
@@ -4248,7 +4257,8 @@
 , O
 und O
 wir O
-L a O
+L O
+a O
 Ciu O
 cf) O
 , O
@@ -6992,7 +7002,8 @@
 deren O
 Helfer O
 ergossen O
-zu haben O
+zu O
+haben O
 , O
 und O
 die O
@@ -7587,7 +7598,7 @@
 - O
 Charlottenburg B-LOC
 l»r O
-die  O
+die O
 Inserate O
 : O
 Robert B-PER
@@ -10662,7 +10673,9 @@
 Uhr O
 . O
 Kreditaktien O
-. . . O
+. O
+. O
+. O
 211 O
 , O
 37 O
@@ -10673,7 +10686,9 @@
 , O
 75 O
 Länderhank O
-. . . O
+. O
+. O
+. O
 — O
 Franzosen O
 100 O
@@ -10688,21 +10703,25 @@
 190 O
 . O
 00 O
-Deutsche Bank B-ORG
-. . O
+Deutsche B-ORG
+Bank I-ORG
+. O
+. O
 249 O
 , O
 62 O
 B O
 . O
 Handelsges. O
-. . O
+. O
+. O
 182 O
 , O
 02 O
 Dresdner B-ORG
 Bank I-ORG
-. . O
+. O
+. O
 102 O
 , O
 87 O
@@ -10713,7 +10732,9 @@
 , O
 63 O
 Nationalbank B-ORG
-. . . O
+. O
+. O
+. O
 13u O
 , O
 12 O
@@ -10827,7 +10848,9 @@
 00 O
 Nordd B-ORG
 Lloyd I-ORG
-. . O
+. O
+. O
+. O
 103 O
 , O
 02 O
@@ -10859,7 +10882,9 @@
 . O
 02 O
 Dortmunder O
-. . . O
+. O
+. O
+. O
 95 O
 . O
 25 O
@@ -10868,7 +10893,9 @@
 . O
 10 O
 Gelsenkirchen B-LOC
-. . O
+. O
+. O
+. O
 220 O
 , O
 25 O
@@ -10877,7 +10904,9 @@
 , O
 30 O
 Consolidation O
-. . O
+. O
+. O
+. O
 — O
 Deutsch-Lux O
 . O
@@ -10906,7 +10935,9 @@
 , O
 75 O
 Dynamit-Trust O
-. . O
+. O
+. O
+. O
 179 O
 , O
 75 O
@@ -11416,7 +11447,8 @@
 Prager O
 Bürgermeister O
 Tr. O
-S r ö' O
+Srö O
+' O
 in O
 Wien B-LOC
 ist O
@@ -11612,7 +11644,8 @@
 . O
 Kleine O
 Chronik O
-- - O
+- O
+- O
 ZugtUsammenstosi O
 int O
 Rcbel O
@@ -11665,7 +11698,8 @@
 schwer O
 beschädigt O
 . O
-— — O
+— O
+— O
 Aus O
 den O
 Vororten O
@@ -12216,11 +12250,15 @@
 Mühle O
 . O
 Mai O
-225.75—226,50 O
+225.75 O
+— O
+226,50 O
 . O
 Roggen O
 inländischer O
-159.00— 100.50 O
+159.00 O
+— O
+100.50 O
 ab O
 Bahn O
 und O
@@ -12246,7 +12284,9 @@
 171,00—180.00 O
 , O
 mittel O
-104.00— 170 O
+104.00 O
+— O
+170 O
 , O
 , O
 gering O
@@ -12367,7 +12407,10 @@
 Bahn O
 , O
 Mai O
-174.75- 173.51! O
+174.75 O
+- O
+173.51 O
+! O
 bis O
 175-176 O
 . O
@@ -12399,7 +12442,8 @@
 M O
 . O
 Preise O
-um 2' O
+um O
+2' O
 /s O
 Uhr O
 . O
@@ -12607,7 +12651,8 @@
 Metnlle O
 . O
 London B-LOC
-. 3 O
+. O
+3 O
 . O
 Januar O
 . O
@@ -12883,7 +12928,7 @@
 einiger O
 Legate O
 über O
-l 1 /* O
+1 O
 Millionen O
 Mark O
 . O
@@ -13101,8 +13146,8 @@
 . O
 Januar O
 Luftdruck O
-m  O
-60  O
+m O
+60 O
 m O
 Seehühe O
 ( O
@@ -13116,30 +13161,38 @@
 . O
 0—10 O
 Luftfeuchtigkeit O
-in  O
+in O
 *7o O
-3.  O
-9  O
-Uhr  O
+3. O
+9 O
+Uhr O
 abends O
-76t,6 1 6,4 O
-SW 3 O
+76t,6 O
+1 O
+6,4 O
+SW O
+3 O
 10 O
 92 O
-4.  O
-7  O
-Uhr  O
+4. O
+7 O
+Uhr O
 vorm. O
-759,8 7,7 O
-W 4 O
+759,8 O
+7,7 O
+W O
+4 O
 10 O
 83 O
-4.  O
-2  O
-Uhr  O
+4. O
+2 O
+Uhr O
 nachm. O
-702,2 | 7,1 O
-WNW 3 O
+702,2 O
+| O
+7,1 O
+WNW O
+3 O
 1 O
 74 O
 3 O
@@ -13331,11 +13384,14 @@
 Graupelschauer O
 . O
 Llaiwncu O
-_ fi O
+_ O
+fi O
 §1 O
-7- L O
+7- O
+L O
 Sä O
-% 2 O
+% O
+2 O
 Windrichtung O
 und O
 Stärke O
@@ -13343,9 +13399,12 @@
 s O
 8 O
 Stationen O
-_ e O
-G ^ O
-3 L O
+_ O
+e O
+G O
+^ O
+3 O
+L O
 Windrichtung O
 und O
 Stärke O
@@ -13365,31 +13424,38 @@
 SSLschwach O
 y O
 bedeckt O
-»U  O
-eit  O
+»U O
+eit O
 um O
-7 * ~t O
-N W mä |,ifl O
+7 O
+* O
+~t O
+NW O
+mä O
+|,ifl O
 wollt. O
 SriU» O
 7i.i O
-.'-.'SW matt. O
+.'-.'SW O
+matt. O
 9 O
 bedeckt O
 Hamburg... O
 71>8 O
-WNW  O
+WNW O
 irtld O
 bcbcdtl O
 Aberdeen B-LOC
 .. O
 770 O
-SW \. O
+SW O
+\. O
 leicht O
 4 O
 tuolfl. O
 Slvinemülibe O
-S12U Irilch O
+S12U O
+Irilch O
 ivvlkig O
 Island O
 .... O
@@ -13399,14 +13465,14 @@
 bedeck» O
 Nngenwalde O
 759 O
-WNW  O
+WNW O
 stürm. O
 l O
 heiter O
 Hfavöer O
 .... O
 7CS O
-WNW  O
+WNW O
 leutt O
 5 O
 wolkig O
@@ -13421,7 +13487,7 @@
 Negern O
 Atemel O
 74'< O
-WSW  O
+WSW O
 steif O
 Ci O
 bedeckt O
@@ -13449,54 +13515,56 @@
 Hannover B-LOC
 ... O
 770 O
-WSW  O
+WSW O
 mau. O
 bedeckt O
 Hapiirmida O
 707 O
-NW  O
+NW O
 leicht O
 -4 O
 bedeckt O
 Berlin B-LOC
-7« JO O
-W mäs:rg O
+7« O
+JO O
+W O
+mäs:rg O
 Reger» O
 Petersburg B-LOC
 . O
 745 O
-SSO  O
+SSO O
 leicht O
 -I O
 bedeckt O
 Dresden B-LOC
 .... O
 7(j9 O
-W  O
+W O
 umhin O
 bedeckt O
 Witna O
 755 O
-SW  O
+SW O
 leicht O
 1 O
 bedeckt O
 Breslau B-LOC
 7t,7 O
-4(1  O
+4(1 O
 schwach O
 bedeut O
 Lelllberg B-LOC
 ... O
 705 O
-W  O
+W O
 Irilch O
 1 O
 Hebet O
 Vromberg B-LOC
 ... O
 760 O
-W  O
+W O
 mjiii O
 7 O
 Rege» O
@@ -13518,11 +13586,12 @@
 lehr O
 leicht O
 8 O
-h. bed. O
+h.bed. O
 ftvauljuvt... O
 SW O
 ichlrach O
-Xiin (I O
+Xiin O
+(I O
 Vlissingen B-LOC
 714 O
 W O
@@ -13574,9 +13643,11 @@
 NO O
 mäßig O
 6 O
-b bed. O
+b O
+bed. O
 teenf O
-NRL 1. O
+NRL O
+1. O
 (eicht O
 -4 O
 wolkt. O
@@ -13588,14 +13659,15 @@
 10 O
 beiter O
 Loitiienankaaiia O
-8 Uhr O
+8 O
+Uhr O
 19 O
 Alm O
 . O
 Moiidaiisgang O
 t O
 Uhr O
-54  O
+54 O
 Min O
 . O
 Vorm O
@@ -13772,7 +13844,7 @@
 je O
 gesehen O
 . O
-50 000 O
+50000 O
 Mark O
 sind O
 für O
@@ -15584,7 +15656,7 @@
 kt O
 rchen» O
 3 O
-.  O
+. O
 April O
 . O
 Die O
@@ -16487,19 +16559,21 @@
 . O
 ( O
 + O
-147 665 O
+147665 O
 ff O
 . O
 ) O
 nngarant O
 . O
 Linie O
-1322 781 O
+1322 O
+781 O
 fl O
 . O
 ( O
 -*- O
-53 626 O
+53 O
+626 O
 fl O
 . O
 ) O
@@ -16519,7 +16593,8 @@
 Ausgabe O
 pro O
 Februar O
-32 056 O
+32 O
+056 O
 Frcs O
 ; O
 Einnahmen O
@@ -16530,7 +16605,8 @@
 bis O
 28 O
 Februar O
-93 130 O
+93 O
+130 O
 Frcs O
 . O
 , O
@@ -16539,7 +16615,8 @@
 die O
 gleiche O
 Zeit O
-69 216 O
+69 O
+216 O
 Frcs O
 . O
 — O
@@ -16549,7 +16626,8 @@
 Betriebs-Einnahmen O
 pro O
 März O
-20 238 O
+20 O
+238 O
 X O
 ( O
 + O
@@ -16579,7 +16657,8 @@
 einen O
 Frachtengewinn O
 von O
-14217 000 O
+14217 O
+000 O
 Kr O
 . O
 erzielt O
@@ -16590,14 +16669,14 @@
 der O
 Betriebskosten O
 von O
-10 499 000 O
+10499000 O
 Kr O
 . O
 ein O
 Itoti O
 gewinn O
 von O
-3 768 000 O
+3768000 O
 Kr O
 . O
 Nach O
@@ -16612,7 +16691,7 @@
 der O
 Reingewinn O
 auf O
-I 170 000 O
+I170000 O
 Kr O
 . O
 , O
@@ -16622,7 +16701,7 @@
 pOL O
 Dividende O
 mit O
-800 000 O
+800000 O
 Kr O
 . O
 verteilt O
@@ -16636,7 +16715,7 @@
 Unterstützungsfonds O
 überwiesen O
 und O
-170 000 O
+170000 O
 Kr O
 . O
 auf O
@@ -16724,14 +16803,14 @@
 März O
 : O
 Potsdamer O
-10 953 O
+10953 O
 Mark O
 . O
 — O
 Allgemeine B-ORG
 Berliner I-ORG
 Omnibus-Aktien-Gesellschaft I-ORG
-169 891 O
+169891 O
 X O
 ( O
 1889 O
@@ -16748,11 +16827,11 @@
 total O
 seit O
 Januar O
-195 379 O
+195379 O
 X O
 ( O
 + O
-11 828 O
+11828 O
 .A! O
 ) O
 . O
@@ -16766,7 +16845,7 @@
 im O
 März O
 1S90 O
-30 942 O
+30942 O
 X O
 ( O
 1890 O
@@ -16805,7 +16884,7 @@
 stellt O
 sieh O
 auf O
-63 335 O
+63335 O
 . O
 # O
 . O
@@ -16845,7 +16924,8 @@
 As O
 ( O
 Mehr-Einnahme O
-333 030.21 O
+333 O
+030.21 O
 Mark O
 ) O
 . O
@@ -16855,7 +16935,8 @@
 auf O
 den O
 Tag O
-36 043.58 O
+36 O
+043.58 O
 X O
 ) O
 . O
@@ -16875,16 +16956,20 @@
 laufenden O
 Geschäftsjahres O
 wurden O
-43 317% O
+43 O
+317 O
+% O
 To O
 . O
 gegen O
-39 084*5, O
+39 O
+084*5, O
 To O
 . O
 im O
 vorigen O
-Jahre verkauft O
+Jahre O
+verkauft O
 , O
 demnach O
 ein O
@@ -16943,7 +17028,8 @@
 verbleibende O
 Gewinnsaldo O
 von O
-485 443 O
+485 O
+443 O
 . O
 /# O
 wie O
@@ -16973,7 +17059,8 @@
 auf O
 neue O
 Rechnung O
-272 063 X O
+272063 O
+X O
 . O
 Dortmunder B-ORG
 DerRbau-Gowellücbnst I-ORG
@@ -16992,7 +17079,7 @@
 durch O
 Ausgabe O
 von O
-666 000 O
+666000 O
 X O
 Vorzugsaktien O
 zu O
@@ -17028,7 +17115,8 @@
 einer O
 neuen O
 Vorzugsaktie O
-5 1200 O
+5 0
+1200 O
 berechtigen O
 . O
 Die O
@@ -17147,7 +17235,8 @@
 einer O
 Dividende O
 von O
-9 pCt O
+9 O
+pCt O
 . O
 vorzuschlagen O
 . O
@@ -17308,13 +17397,17 @@
 Effecten-Smretäi O
 . O
 Kreditaktien O
-255 1 /? O
+255 O
+1 O
+/? O
 , O
 Franzosen O
 184V« O
 , O
 Lombarden O
-102 7 /s- O
+102 O
+7 O
+/s- O
 Galizier O
 lsiä'A O
 , O
@@ -17408,7 +17501,8 @@
 217,00 O
 , O
 Lombarden O
-122 00 O
+122 O
+00 O
 . O
 Galizier O
 194.00 O
@@ -17417,7 +17511,8 @@
 201.00 O
 , O
 Elhetalbahn O
-215 75 O
+215 O
+75 O
 . O
 österr O
 . O
@@ -17460,7 +17555,8 @@
 24050 O
 . O
 Länderbank O
-216 50 O
+216 O
+50 O
 , O
 Alpine O
 Montan O
@@ -17483,7 +17579,9 @@
 105 O
 , O
 Englische O
-2 3 / O
+2 O
+3 O
+/ O
 , O
 proc O
 . O
@@ -17512,9 +17610,12 @@
 94V« O
 , O
 Italiener O
-9I- 1 /« O
+9I- O
+1 O
+/« O
 , O
-4procent O
+4 O
+procent O
 . O
 ungarische O
 Goldrente O
@@ -17541,7 +17642,8 @@
 . O
 , O
 Spanier O
-— — O
+— O
+— O
 , O
 Canada O
 Pacific O
@@ -17752,7 +17854,8 @@
 . O
 per O
 Juni-Juli O
-190 00 O
+190 O
+00 O
 , O
 per O
 September O
@@ -17785,7 +17888,8 @@
 September O
 - O
 Oktober O
-155 00 O
+155 O
+00 O
 . O
 — O
 Pommerscher O
@@ -17834,7 +17938,8 @@
 September O
 - O
 Oktober O
-34 80 O
+34 O
+80 O
 . O
 — O
 Petroleum O
@@ -17988,7 +18093,8 @@
 pr O
 . O
 Mai O
-17 37 O
+17 O
+37 O
 , O
 pr O
 . O
@@ -19719,7 +19825,7 @@
 die O
 Höhe O
 von O
-1 825 147 O
+1825147 O
 , O
 04 O
 M O
@@ -20049,7 +20155,8 @@
 der O
 Pester O
 Filiale O
-80 999 O
+80 O
+999 O
 , O
 4.' O
 ! O
@@ -21482,7 +21589,8 @@
 Tafts B-PER
 . O
 DO O
-New York B-LOC
+New B-LOC
+York I-LOC
 . O
 12 O
 . O
@@ -22818,7 +22926,8 @@
 mitzuarbeiten O
 . O
 Stimmenverluste O
-der Marxisten O
+der O
+Marxisten O
 . O
 Das O
 Ergebnis O
@@ -22899,9 +23008,9 @@
 Insgesamt O
 wurden O
 von O
-14 750 O
+14750 O
 Wahlberechtigten O
-13 754 O
+13754 O
 Stimmen O
 abgegeben O
 . O
@@ -22984,7 +23093,7 @@
 stattgegeben O
 worden O
 . O
-w-mTmr»nT-TTni[r —— — ,, | , «im— O
+... O
 Zigarettenfabrik O
 Nestor B-PER
 Gianaclis I-PER
@@ -26113,7 +26222,8 @@
 Eisenbahn O
 - O
 Betriebs O
-- Inspektor O
+- O
+Inspektor O
 Beemelman» O
 in O
 Straßburg B-LOC
@@ -29496,12 +29606,17 @@
 und O
 groben O
 Eisenwaaren O
-. . O
+. O
+. O
+. O
 242,746 O
 3,858,602 O
 feinen O
 Eisenwaaren O
-. . . . O
+. O
+. O
+. O
+. O
 3,401 O
 33,025 O
 Maschinen O
@@ -29839,7 +29954,9 @@
 vorgearbeitet O
 oder O
 zerkleinert O
-. . . O
+. O
+. O
+. O
 86,560 O
 129,425 O
 zusammen O
@@ -29858,7 +29975,9 @@
 gesägt O
 jc O
 . O
-. . . O
+. O
+. O
+. O
 412,613 O
 243,786 O
 zusammen O
@@ -30237,12 +30356,14 @@
 Rollendor O
 4788—88 O
 , O
-5980 —88 O
+5980 O
+— O
+88 O
 . O
 Post'check O
 Berlin B-LOC
 Ott O
-107 941 O
+107941 O
 Die O
 „ O
 DAZ O
@@ -30963,8 +31084,10 @@
 schlägt O
 den O
 Ein- O
-w ***" O
-v “ O
+w O
+***" O
+v O
+“ O
 — O
 11 O
 — O
@@ -30973,7 +31096,9 @@
 1 O
 Dicht O
 , O
-J ch.ru er O
+J O
+ch.ru O
+er O
 jlebeac O
 der O
 fremden O
@@ -32901,8 +33026,16 @@
 10 O
 . O
 ;lg O
-i n O
-,. r. b c r i (!) t.) O
+i O
+n O
+,. O
+r. O
+b O
+c O
+r O
+i O
+(!) O
+t.) O
 Nach O
 dem O
 „ O
@@ -33228,8 +33361,11 @@
 Führer O
 . O
 " O
-Philipp O
-Mi 11 c t O
+Philipp B-PER
+Mi I-PER
+11 I-PER
+c I-PER
+t O
 z O
 . O
 B O
@@ -33494,7 +33630,7 @@
 Zijser O
 angesichts O
 von O
-150 000 O
+150000 O
 Eisenbahnern O
 , O
 die O
@@ -33897,7 +34033,8 @@
 geben O
 , O
 fcifo O
-Me >clieusrage O
+Me O
+>clieusrage O
 dem O
 Internationalen B-ORG
 Gerichtshof I-ORG
@@ -42745,7 +42882,7 @@
 begonnen O
 . O
 Auf O
-Broiii8law O 
+Broiii8law O
 Hubermanns B-PER
 Wiedergabe O
 von O
@@ -43661,7 +43798,7 @@
 Jacobthals I-PER
 Violinspiel O
 . O
-Kühn O 
+Kühn O
 stellte O
 er O
 in O
@@ -43731,7 +43868,7 @@
 allem O
 aber O
 muß O
-Fräulein O 
+Fräulein O
 Sternberg B-PER
 noch O
 lernen O
@@ -44145,7 +44282,7 @@
 Malz O
 trotz O
 des O
-Ausfuhrverbots  O
+Ausfuhrverbots O
 nach O
 Norddeutschland B-LOC
 gebracht O
